tests/php/validation/test-class-amp-validation-manager.php

```diff
@@ -658,23 +685,70 @@ static function() {
 		$this->assertEmpty( $field['results'] );
 	}
 
+	/**
+	 * Test map_meta_cap.
+	 *
+	 * @covers AMP_Validation_Manager::map_meta_cap()
+	 */
+	public function test_map_meta_cap() {
+		$this->assertEquals(
+			AMP_Validation_Manager::map_meta_cap( [ AMP_Validation_Manager::VALIDATE_CAPABILITY ], AMP_Validation_Manager::VALIDATE_CAPABILITY ),
+			[ 'manage_options' ]
+		);
+
+		$this->assertEquals(
+			AMP_Validation_Manager::map_meta_cap( [ 'install_plugins', AMP_Validation_Manager::VALIDATE_CAPABILITY, 'update_core' ], AMP_Validation_Manager::VALIDATE_CAPABILITY ),
+			[ 'install_plugins', 'manage_options', 'update_core' ]
+		);
```

Comment:
Shouldn't these assert parameters be the other way around?

Reply:
Fixed in edfda942618f413a578a5cc9e985f69d88d7ec19